explore/src/main/scala/explore/targeteditor/AsterismEditor.scala

```diff
@@ -123,22 +111,28 @@ object AsterismEditor {
         React.Fragment(
           props.renderInTitle(
             TargetSelectionPopup(
-              trigger = Reuse.by(adding.value)(
+              trigger = Reuse.by(adding.get)(
```

Comment:
We can use `adding.map( a => Button(...))` now.

Reply:
Ah, yeah. nice.